emote/callback.py

```diff
@@ -211,7 +211,7 @@ def end_training(self, *args, **kwargs):
     def state_dict(self) -> Dict[str, Any]:
         return {}
 
-    def load_state_dict(self, state_dict: Dict[str, Any]):
+    def load_state_dict(self, state_dict: Dict[str, Any], *args, **kwargs):
```

Comment:
I don't think this has to be generic! We have previously used three different flags and I'd prefer to maintain that flow:

* load_weights
* load_optimizers
* load_hparams - this is a bit badly named, but essentially inf-steps and other things like that that aren't in either of the above categories! 

I think that would make more sense than having flags that are "only x". What if I want only optimizers? Only schedules? Weights and optimizers but no schedules? :-)